plugins/woocommerce/readme.txt

```diff
@@ -8,7 +8,7 @@ Stable tag: 7.6.1
 License: GPLv3
 License URI: https://www.gnu.org/licenses/gpl-3.0.html
 
-WooCommerce is the world’s most popular open-source eCommerce solution.
+WooCommerce has helped millions of merchants build successful businesses for the long term. Get everything you need to launch an online store in days and keep it growing for years. From your first sale to millions in revenue, Woo is with you.
```

Comment:
Text is lovely, but wanted to note that at 242 chars (by comparison, the current description is 71 chars long) this exceeds the [recommended limited of 150 chars](https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/). The possible risk being, the text may be truncated in some situations.

As long as we're fine with that, this is good to merge (wanted to pause and check we're aware of the recommendation first, though).

Reply:
Thanks, wasn't aware of it. Updated after consultation with @dlcallaway in [cd8dcf2](https://github.com/woocommerce/woocommerce/pull/38074/commits/cd8dcf29a711492f4029c1c977dd1a2c827a87bd)